spec/lib/mcb/commands/courses/list_spec.rb

```diff
@@ -11,75 +11,71 @@
     let(:provider2) { create(:provider, recruitment_cycle: current_cycle) }
     let(:provider3) { create(:provider, recruitment_cycle: additional_cycle) }
 
-    let(:course1) { create(:course, provider: provider1) }
-    let(:course2) { create(:course, provider: provider2) }
-    let(:course3) { create(:course, provider: provider3) }
+    let(:course_present) { create(:course, provider: provider1, name: 'First present course') }
+    let(:course_present2) { create(:course, provider: provider2, name: 'Second present course') }
+    let(:course_not_present) { create(:course, provider: provider3, name: 'Non-present course') }
 
     it 'lists all courses for the default recruitment cycle' do
-      course1
-      course2
-      course3
+      course_present
+      course_not_present
 
       command_output = list.execute[:stdout]
 
-      expect(command_output).to include(course1.course_code)
-      expect(command_output).to include(course1.name)
-
-      expect(command_output).to include(course2.course_code)
-      expect(command_output).to include(course2.name)
```

Comment:
I think the reason this was here originally is to prove that the list contains _all_ the courses, by default (as per the name of the test). Was there a specific reason you removed it?

Reply:
Adding it back! Mistook it to look like it tested the same thing twice.